### ISSUE TITLE
[Bug]: Navigation bar URL input field gets stuck after inputting something that is not a web address

### ISSUE THREAD

**Issue by dehyde** [getferdi/ferdi#2257]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/getferdi/ferdi/blob/develop/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/getferdi/ferdi/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/getferdi/ferdi/issues) for a feature request that matches the one I want to file, without success.

### Ferdi Version

5.6.3

### What operating system are you using?

Windows

### Server

Ferdi

### Operating System Version

11

### What arch are you using?

x64

### Steps to reproduce

1. Go to Navigation bar URL input field
2. Enter text that is not a URL
3. Press enter
4. Try to edit the text inside the URL input field

### Expected Behavior

A google search result for that input string (was working on a previous version)

### Actual Behavior

URL field returns to previous text, is not editable until clicking outside of field and clicking inside again

### Screenshots

https://www.loom.com/share/0865282d7d7f4079938348a40ae910f7

**Comment by vraravam** [comment]
I don't think that the non-url would automagically convert to a google search - ever in Ferdi.

That being said, the only changes to the url-handling was an upgrade of a dependency library that validates url. I'm not sure if this behavior can be reverted.

If there's traction for converting a non-http/https url to a search (based on the user preferences), that would be welcome as a contribution.

OTOH, I also want to caution that these kind of features take Ferdi closer to a default browser - which we are not aiming to be.
